en/micro-integrator/docs/setup/brokers/configure-with-ActiveMQ.md

```diff
@@ -160,7 +160,7 @@ Let's see how some of the key concepts of system security such as authentication
 |------------------------------------------------|----------------------------------------------------------------|
 | [Authentication](#authentication) | Simple authentication and JAAS plugins.                                     |
 | [Authorization](#authorization) | Built-in authorization mechanism using XML configuration.                     |
-| [Availability](#availability)  | Master/Slave configurations using fail-over transport in ActiveMQ (not to be confused with the Micro Integrator's transports). |
+| [Availability](#availability)  | primary/secondary configurations using fail-over transport in ActiveMQ (not to be confused with the Micro Integrator's transports). |
```

Comment:
Why is primary/secondary not starting with uppercase letters here? Can you check all other places for such cases.

Reply:
Will check and fix in a fresh PR. There are a couple of other instances of master that we are checking as well since they are part of API calls.

Reply:
Fixed with the following PRs.

https://github.com/wso2/docs-ei/pull/1715
https://github.com/wso2/docs-ei/pull/1714